### DISCUSSION TITLE
EM540 serial connection and serialbattdry

### DISCUSSION THREAD

**Discussion by bigblue1735** [Louisvdw/dbus-serialbattery#734] [Q&A]
Hi,
I am using serialbattery several times without problem. But on my last installation I first installed the serialbattery plugin. The plugin was working well. Afterwards I connected the EM540 with another serial adapter but the Cerbo did not find the Grid meter. I rebooted several time try to chance everything without success.

After I removed the serialbattery the Gridmeter was found directly and the system is now running fine. Now I want to have a look what will be the problem because I still want to habe both connected to the system. Is it possible to bind serialbattery to one special ttyusb device?

**Comment by mr-manuel** [comment]
Since you are not providing any useful information like the driver version you are using, Venus OS version and so on I can only guess.

Install the nightly from the dev branch and it should work, since there were some checks added that prevent wrong device recognition.
